polkadot/runtime/parachains/src/paras_inherent/mod.rs

```diff
@@ -333,6 +331,27 @@ impl<T: Config> Pallet<T> {
 		let now = frame_system::Pallet::<T>::block_number();
 		let config = configuration::ActiveConfig::<T>::get();
 
+		// Before anything else, update the allowed relay-parents.
+		{
+			let parent_number = now - One::one();
```

Comment:
```suggestion
			let parent_number = parent_header.number();
```

Reply:
this doesn't work for some reason

Reply:
I tried doing this and a bunch of unit tests started failing because they aren't properly populating the parent header. Fixing this is pretty tedious so I'm inclined to leave this as is since it's been this way since forever and it's only a cosmeting change 😄